manifests/piped/values.yaml

```diff
@@ -86,3 +86,6 @@ resources: {}
 
 serviceAccount:
   annotations: {}
+
+# Specifies whether to restrict Piped access to within Namespace.
+useClusterRole: true
```

Comment:
I am expecting something like this:

``` yaml
serviceAccount:
   create: true
   name: ... # specify an external service account to use and set create to be false

rbac:
   create: true
   scope: cluster # this can be cluster or namespace
   rules: [] # list of rules if they want to customize it
```

Reply:
Looks good! I feel like `serviceAccount.create` isn't needed (Just give `serviceAccount.name` then it will be true automatically); I'll do that.

Reply:
`create` is added to be unified. And not only `name` we also have `annotations` and other fields so adding `create` makes them become clear.

Reply:
Okay. I'll do so.